<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>bgLogin.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(74, 204, 253);
font-family: Daytona;
font-size : 50px;


</string>
   </property>
   <property name="text">
    <string>Delete Profile</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>620</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>640</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  display: block;
  font-size: 10px;
  font-family: sans-serif;
  text-decoration: none;
  color: #ddd;
  border: 2px solid #ddd;
  padding: 10px;
  transition: all .25s;
  border-radius:10px;
}
QPushButton:hover{
 	 border-color:rgb(250, 113, 103);
	color: #FFF;
	background:rgb(250, 113, 103);
	letter-spacing:1px;
	font-weight : bold;
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(74, 204, 253);
font-family: Daytona;
font-size : 25px;
</string>
   </property>
   <property name="text">
    <string>Please enter the ID of the profile you wish to delete  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="profileID">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>260</y>
     <width>420</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
font-size : 15px;
color  : black ;

}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnMenu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  display: block;
  font-size: 10px;
  font-family: sans-serif;
  text-decoration: none;
  color: #ddd;
  border: 2px solid #ddd;
  padding: 10px;
  transition: all .25s;
  border-radius:10px;
}
QPushButton:hover{
 	 border-color:rgb(74, 204, 253);
	color: rgb(74, 204, 253);
	letter-spacing:1px;
	font-weight : bold;
}</string>
   </property>
   <property name="text">
    <string>&lt;&lt;retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
